doc/cephadm/operations.rst

```diff
@@ -402,18 +402,24 @@ identifying anomalies.
 
 Client keyrings and configs
 ===========================
-
 Cephadm can distribute copies of the ``ceph.conf`` file and client keyring
-files to hosts. It is usually a good idea to store a copy of the config and
-``client.admin`` keyring on any host used to administer the cluster via the
-CLI.  By default, cephadm does this for any nodes that have the ``_admin``
-label (which normally includes the bootstrap host).
+files to hosts. Starting from versions 16.2.10 (Pacific) and 17.2.1 (Quincy),
+in addition to the default location ``/etc/ceph/`` cephadm also stores config
+and keyring files in the ``/var/lib/ceph/<fsid>/config`` directory. It is usually
+a good idea to store a copy of the config and ``client.admin`` keyring on any host
+used to administer the cluster via the CLI. By default, cephadm does this for any
+nodes that have the ``_admin`` label (which normally includes the bootstrap host).
+
+.. note:: the new configuration location ``/var/lib/ceph/<fsid>/config`` is used by
+  cephadm only. Ceph daemons will still use files on ``/etc/ceph/``.
```

Comment:
can we add to this note why we also store a copy in `/var/lib/ceph/<fsid>/config`? Mainly, that it helps cephadm infer what config to use since it being in the fsid dir guarantees it's associated with this cluster?

Reply:
Agreed, is the order in https://github.com/ceph/ceph/pull/46369#discussion_r879682390 documented elsewhere?

I can see people shooting themselves in the foot because they don't know where the canonical copy is, that will overwrite `/etc/ceph` files that their automation populates.

Reply:
@anthonyeleven right now is only documented on the cephadm shell section since it only affects this part (where cephadm shell loads the config). As stated in the note: this doesn't affect the configuration used by ceph daemons. 

To make it clear from where the config is taken, cephadm (as in the following example) shows the location when starting the shell. So the user normally should be aware from where the config is loaded:

```
[root@ceph-node-0 ~]# cephadm shell
Inferring fsid d87d598a-ebd2-11ec-b9b0-525400581dfe
Inferring config /var/lib/ceph/d87d598a-ebd2-11ec-b9b0-525400581dfe/mon.ceph-node-0/config
Using ceph image with id '08b9d8a1fe1b' and tag 'custom-v0.14' created on 2022-06-03 14:48:17 +0000 UTC
registry.hub.docker.com/rkachach/ceph@sha256:9933fba86fe39e492d04b6551c79e129855a003580629650fda17142b5b43907

```

Anyway, please suggest any additional suitable location to put this doc into in case you feel it would be helpful for end users.